submissions/drewzag/task_js_dom/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/styles.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <title>task_js_dom</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav class="nav">
+        <ul class="list"></ul>
+        <div class="menu__icon">
```

Comment:
The menu button should be square. in your case it full width [watch video](https://share.vidyard.com/watch/s6Sm5Sd6ohHCafpEFZ7tx3?)